src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,71 @@
+package lotto.view;
+
+import lotto.domain.EarningRate;
+import lotto.domain.LottoNumber;
+import lotto.domain.LottoPrize;
+import lotto.domain.LottoStatistics;
+import lotto.domain.LottoTickets;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+public class OutputView {
+    private static final String PROFIT = "이득";
+    private static final String LOSS = "손해";
+
+    public static void printTickets(LottoTickets lottoTickets) {
+        System.out.printf("%s개를 구매했습니다.%n", lottoTickets.size());
+        System.out.println(lottoTickets);
+
+        lottoTickets.getLottoTickets().forEach(lottoTicket ->
+                printNumbers(lottoTicket.getLottoNumbers().getLottoNumbers())
+        );
+    }
+
+    private static void printNumbers(List<LottoNumber> lottoNumbers) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        stringBuilder.append("[");
+
+        for (int i = 0; i < lottoNumbers.size(); i++) {
+            if (i != 0) {
+                stringBuilder.append(", ");
+            }
+
+            stringBuilder.append(lottoNumbers.get(i));
+        }
+
+        stringBuilder.append("]");
+
+        System.out.println(stringBuilder);
```

Comment:
`lottoNumbers`를 그대로 출력해도 동일한 결과를 얻을 수 있을 것 같아요.
```suggestion
        System.out.println(lottoNumbers);
````